src/databricks/labs/remorph/lineage/execute.py

```diff
@@ -0,0 +1,51 @@
+import datetime
+import logging
+from pathlib import Path
+
+from databricks.labs.remorph.intermediate.dag import DAG
+from databricks.labs.remorph.intermediate.root_tables import RootTableIdentifier
+
+logger = logging.getLogger(__name__)
+
+
+def generate_cot_file(dag: DAG) -> str:
+    lineage_str = "flowchart TD\n"
+    for node_name, node in dag.nodes.items():
+        if node.parents:
+            for parent in node.parents:
+                lineage_str += f"    {node_name.capitalize()} --> {parent.name.capitalize()}\n"
+        else:
+            # Include nodes without parents to ensure they appear in the diagram
+            lineage_str += f"    {node_name.capitalize()}\n"
+    return lineage_str
+
+
+def write_lineage_file(output_folder, lineage_file_content) -> None:
```

Comment:
there will be file_utils, if none exists create one inside file_utils for writing the output.